base/io.jl

```diff
@@ -1003,6 +1000,46 @@ eltype(::Type{<:EachLine}) = String
 
 IteratorSize(::Type{<:EachLine}) = SizeUnknown()
 
+struct EachOfIO{T, IOT <: IO}
```

Comment:
It'd make sense to rename `EachOfIO` too.

Reply:
Maybe `ReadEachIterator`?

Reply:
Good catch. Thanks.